### PR TITLE
Unify bitmap and bitmapmask

### PR DESCRIPTION
**Pull request by tehKaiN**
Related to issue #12

### CODE REVIEW

**Comment by tehKaiN** [comment]
bitmap_conv is still a mess but I'll leave it for some other time.

File: tools/bitmap_conv/bitmap_conv.c [hunk 1]
```diff
@@ -206,24 +206,40 @@ void writeMask(
...
 	uint32_t ulPos;
 	FILE *pOut;
-	pOut = fopen(g_szMaskOutputPath, "wb");
+
+	if(uwWidth & 0xF) {
+		printf("Width is not divisible by 16!\n");
+		return;
+	}
+
+	pOut	= fopen(g_szMaskOutputPath, "wb");
```
Comment (approxit) [inline tools/bitmap_conv/bitmap_conv.c]: Tab...?

Comment (tehKaiN) [reply tools/bitmap_conv/bitmap_conv.c]: Whoops, fixed!

**Comment by approxit** [comment]
pOut file is a little misleading... But rest is LGTM.

**Comment by tehKaiN** [comment]
Now better?

**Comment by approxit** [comment]
LGTM

**Pull request merged**